web/html/src/components/input/InputBase.js

```diff
@@ -31,6 +33,7 @@ type State = {
 
 class InputBase extends React.Component<Props, State> {
   static defaultProps = {
+    defaultValue: undefined,
```

Comment:
this isn’t related with this PR but I think we don’t need any of these defaultprops. Not passing a prop is the same as having it undefined

Reply:
Without them flow complains that optional properties need to have a default.